docs/quickstart.md

```diff
@@ -293,6 +293,985 @@ hive>
+*    |_|  |_|\___/ \___/ \__,_|_|\___|     *
+*                                          *
+============================================
+
+Welcome to Hoodie CLI. Please type help if you are looking for help.
+
+#Now create Copy On Write (COW) Hudi data set 
+hoodie->create --path /user/hive/warehouse/stock_ticks_cow --tableName stock_ticks_cow --tableType COPY_ON_WRITE
+.....
+18/09/24 22:18:35 INFO table.HoodieTableMetaClient: Finished Loading Table of type COPY_ON_WRITE from /user/hive/warehouse/stock_ticks_cow
+Metadata for table stock_ticks_cow loaded
+
+#Now create Merge On Read (MOR) Hudi data sets
+hoodie:stock_ticks_cow->create --path /user/hive/warehouse/stock_ticks_mor --tableName stock_ticks_mor --tableType MERGE_ON_READ
+....
+18/09/24 22:18:54 INFO table.HoodieTableMetaClient: Finished Loading Table of type MERGE_ON_READ from /user/hive/warehouse/stock_ticks_mor
+Metadata for table stock_ticks_mor loaded
+exit
+exit
+```
+You can use HDFS web-browser to look at the datasets 
+`http://namenode:50070/explorer.html#/user/hive/warehouse/stock_ticks_cow`. You will notice that there is only `.hoodie`
+directory present and the dataset is empty. 
+
+There will be a similar setup when you browse the MOR dataset
+`http://namenode:50070/explorer.html#/user/hive/warehouse/stock_ticks_mor`
+
+
+#### Step 3: Incrementally ingest data from Kafka topic
+
+Hudi comes with a tool named DeltaStreamer. This tool can connect to variety of data sources (including Kafka) to
+pull changes and apply to Hudi dataset using upsert/insert primitives. Here, we will use the tool to download
+json data from kafka topic and ingest to both COW and MOR tables we initialized in the previous step.
+
+```
+docker exec -it adhoc-2 /bin/bash
+
+# Run the following spark-submit command to execute the delta-streamer and ingest to stock_ticks_cow dataset in HDFS
+spark-submit --class com.uber.hoodie.utilities.deltastreamer.HoodieDeltaStreamer /var/hoodie/ws/hoodie-utilities/target/hoodie-utilities-0.4.4-SNAPSHOT.jar --source-class com.uber.hoodie.utilities.sources.JsonKafkaSource --source-ordering-field ts  --target-base-path /user/hive/warehouse/stock_ticks_cow --target-table stock_ticks_cow --props /var/demo/config/kafka-source.properties
+....
+....
+2018-09-24 22:20:00 INFO  OutputCommitCoordinator$OutputCommitCoordinatorEndpoint:54 - OutputCommitCoordinator stopped!
+2018-09-24 22:20:00 INFO  SparkContext:54 - Successfully stopped SparkContext
+# Run the following spark-submit command to execute the delta-streamer and ingest to stock_ticks_mor dataset in HDFS
+spark-submit --class com.uber.hoodie.utilities.deltastreamer.HoodieDeltaStreamer /var/hoodie/ws/hoodie-utilities/target/hoodie-utilities-0.4.4-SNAPSHOT.jar --source-class com.uber.hoodie.utilities.sources.JsonKafkaSource --source-ordering-field ts  --target-base-path /user/hive/warehouse/stock_ticks_mor --target-table stock_ticks_mor --props /var/demo/config/kafka-source.properties
```

Comment:
this is now `hoodie-utilities-0.4.4-SNAPSHOT.jar` with the new release. same comment on other 3 occurrances as well.. Can we name the jar differently inside the container without the version? that way it will continue to work? (I understand it takes away knowing what version is being tested.) thoughts?

Reply:
Also DeltaStreamer now takes a `--storage-type COPY_ON_WRITE/MERGE_ON_READ` argument which is required.. we can use this and get rid of steps to create the dataset manually via CLI? DeltaStreamer will create the dataset if the basePath does not exist.

Reply:
@bvaradar https://github.com/docker/compose/issues/3574 don't think `compose up` will pull in latest images.. I am going remove all containers and retry.. We probably need to have a better support in the script for optionally also doing a `compose pull` before?

Reply:
That worked @bvaradar .. One more issue I found was between recreating containers, the path on host machine `/tmp/hadoop*` needs to be blown away.. Can we add a `rm -rf` before the `mkdir -p` in the setup script.. other than that, verified that building and deltastreamer work..